_data/contracts.yml

```diff
@@ -1,12 +1,12 @@
 kovan:
   registry: "0xe331bf20044a5b24c1a744abc90c1fd711d2c08d"
   mpe: "0x39f31ac7b393fe2c6660b95b878feb16ea8f3156"
```

Comment:
Have to update the addresses as well.

Reply:
@anandrgitnirman Kindly update the address